js/packages/mobile-wallet-adapter-protocol/src/parseHelloRsp.ts

```diff
@@ -14,7 +16,7 @@ export default async function parseHelloRsp(
         crypto.subtle.exportKey('raw', associationPublicKey),
         crypto.subtle.importKey(
             'raw',
-            payloadBuffer,
+            payloadBuffer.slice(0, ENCODED_PUBLIC_KEY_LENGTH_BYTES),
```

Comment:
This change is necessary because the response buffer now contains more than just the public key. It now includes also `session_properties`

Reply:
yes, well it might include the session properties. Subtle crypto was throwing an error if I did not slice off the extra bytes. 

if the session properties are not present (ie when interacting with a legacy wallet), this is still safe.